Code/Tools/FBuild/FBuildCore/Protocol/Protocol.h

```diff
@@ -31,7 +31,7 @@ namespace Protocol
     enum : uint16_t { PROTOCOL_PORT = 31264 }; // Arbitrarily chosen port
 
     // Protocol Version
-    enum : uint32_t { PROTOCOL_VERSION_MAJOR = 22 };    // Changes here make workers incompatible
+    enum : uint32_t { PROTOCOL_VERSION_MAJOR = 23 };    // Changes here make workers incompatible
```

Comment:
The protocol generally needs to remain compatible between versions because many users rely on this (sharing workers between various projects on varying versions). The major protocol version would ideally never be bumped, but exists for should a change be needed in the future that is impossible to keep compatible.

The minor version can be used to allow workers and clients to operate in a forwards or backwards compatible fashion. There isn't an example of this exact type of operation, but I think GetDistributableJobToProcess could be modified to pas in the minor version of the worker requesting the job which it could use to know if that worker can support a job with this flag.

I hope to do further cleanup in the future that would make this logic more easily exist in the CompilerDriver which would make a change list this simpler.